### ISSUE TITLE
DeepScence function signature

### ISSUE THREAD

**Issue by wangjiaoman1999** [anthony-qu/DeepScence#5]
Hello, I noticed that the DeepScence function signature does not contain a species="mouse" argument. How should I adjust the parameters when my data come from mouse single-cell sequencing?
DeepScence function signature:
(adata, lambda_ortho=0.1, hidden_sizes=(32,), batchnorm=False, dropout=0.5, epochs=300, validation_split=0.1, reduce_lr=10, early_stop=25, batch_size=None, learning_rate=0.005, n=5, direction_n=5, random_state=0, verbose=False, binarize=False, denoise=True)

**Comment by anthony-qu** [comment]
Hi, thank you for the feedback! 

I believe there is a parameter called species if you download the main branch (as instructed in the readme). Also, if you look into api.py, you'll see that DeepScence function takes as input a string for species (with default = "human", and you can set it to "mouse"). Can you try running DeepScence with `species = "mouse"` added as an additional parameter and let me know of the output, please?
